community/examples/AMD/hpc-cluster-amd-slurmv5.yaml

```diff
@@ -65,20 +65,46 @@ deployment_groups:
       - type: shell
         source: modules/startup-script/examples/install_ansible.sh
         destination: install_ansible.sh
+      - $(swfs.install_nfs_client_runner)
+      - $(swfs.mount_runner)
       - $(spack.install_spack_deps_runner)
       - $(spack.install_spack_runner)
       # the following installation of AOCC may be automated in the future
       # with a clear direction to the user to read the EULA at
       # https://developer.amd.com/aocc-compiler-eula/
+      - type: shell
+        content: "shutdown -h +1"
```

Comment:
Is there a reason for the +1m? (I imagine it doesn't hurt, just curious)

Reply:
I had this discussion with @nick-stroud. Empirically, this forks into the background and allows the startup script to complete. I like the idea of allowing the SystemD unit running the startup script to complete properly rather than potentially being shutdown prematurely when SystemD is triggered to shutdown.

I wish we could do `+15s` or similar, but 1 minute seems fine.

Reply:
Thanks for the context, that makes sense